content/header/contents.lr

```diff
@@ -0,0 +1,13 @@
+_hidden: yes
+---
+title: Slova Nova
+---
+subtitle: Biblical resources for house-church workers around the world
+---
+hero_css: is-light is-bold
+---
+button_txt: Fund this Project
+---
+button_url: https://slova.givingfuel.com/support
+---
+button_css: is-info is-large
```

Comment:
I did not keep a consistent naming pattern but any of these "css" attributes encountered are intended to be a convenient spot to inject [Bulma](https://bulma.io/documentation/) class names.